UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs

```diff
@@ -96,17 +96,17 @@ public struct TypeAndStrength
 				}
 			}
 
-			BodyReactionAmount = ProcessingAmount * ReagentMetabolismMultiplier;
+			BodyReactionAmount = ProcessingAmount;
 		}
 
 		foreach (var bodyPart in Toloop)
 		{
-			var Individual = bodyPart.ReagentMetabolism * bodyPart.BloodThroughput * bodyPart.CurrentBloodSaturation * Mathf.Max(0.10f, bodyPart.RelatedPart.TotalModified) * ReagentMetabolismMultiplier;
+			var Individual = bodyPart.ReagentMetabolism * bodyPart.BloodThroughput * bodyPart.CurrentBloodSaturation;
```

Comment:
why remvoe the second multiplication?

Reply:
what gilles asked

Reply:
to make it so body part  Damage doesn't slow down medical chemistry, E.g scenarios where you're trying to heal heavily damaged limb but it's damage so the chemistry doesn't work so well